assets/css/custom.scss

```
@@ -3908,3 +3908,1439 @@ font-size: 1.55rem;
+      margin-bottom: 0.6rem;
+      padding-bottom: 0.4rem;
+      letter-spacing: -0.01em;
+      position: relative;
+
+      &::before {
+        content: "";
+        display: block;
+        width: 56px;
+        height: 4px;
+        @include cs-brand-gradient;
+        margin-bottom: 1.1rem;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)